library/alloc/src/collections/btree/node.rs

```diff
@@ -120,11 +119,11 @@ struct BoxedNode<K, V> {
 
 impl<K, V> BoxedNode<K, V> {
     fn from_leaf(node: Box<LeafNode<K, V>>) -> Self {
-        BoxedNode { ptr: Box::into_unique(node) }
+        BoxedNode { ptr: Unique::from(Box::leak(node)) }
```

Comment:
This should not be needed with the fixed `into_unique`.

Reply:
But the unstable note on `into_unique` says that's what we should use? The `.into()` also works in this case, but not for `from_internal`.

In any case, it clearly does not help to avoid the change to unique.

Reply:
> But the unstable note on into_unique says that's what we should use?

Oh, I didn't know that. Sure then.